### ISSUE TITLE
TmodLoader 1.4 launching Terraria, but Terraria is not rendering [No Errors] [Reinstalling does not work]

### ISSUE THREAD

**Issue by Celestaros** [bluemagic123/tModLoader#2625]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

Launching TmodLoader seem to work, no crash during loading. Once the loading of tModLoader is finished however launching Terraria 1.4, the game launches for Steam without actually rendering or being present in the Task Manager. Nothing else happens, Steam keep thinking the game is running and no crashes happens either.

### Log File

[client2.log](https://github.com/tModLoader/tModLoader/files/8985548/client2.log)

### Steps to reproduce

1. Launching TmodLoader.

### Expected Behavior

Actually running the game instead of it not appearing in the task manager or rendering on the screen

### Actual Behavior

Steam says the game is running, but nothing happens

### Reproduction frequency

All the time.

**Comment by Rijam** [comment]
If I understand your description correctly, this is intended behavior. Re-Logic requested that playing tModLoader also counted as play time for Terraria.

**Comment by JavidPack** [comment]
This issue has either no recent activity or no reproducible steps, so it is being closed. We assume this issue has solved itself in some manner. Feel free to open a new issue if the issue persists and fill out the issue template completely.
